charts/argo-cd/Chart.yaml

```
@@ -22,3 +22,4 @@ dependencies:
 annotations:
   artifacthub.io/changes: |
     - "[Fixed]: Correct ArgoCD notification subscriptions type"
+    - "[Fixed]: Correct ArgoCD applicationset controller port"
```

```suggestion
    - "[Fixed]: Correct ArgoCD applicationset controller port"
```
The old line needs to be replaced/removed